NuGet.config

```diff
@@ -13,7 +13,6 @@
     <!--  Begin: Package sources from dotnet-runtime -->
     <!--  End: Package sources from dotnet-runtime -->
     <!--  Begin: Package sources from dotnet-templating -->
-    <add key="darc-pub-dotnet-templating-f6749ec0" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-templating-f6749ec0/nuget/v3/index.json" />
```

Comment:
@mmitche we're getting the missing 6.0.2 sources error again (that we originally got in https://github.com/dotnet/sdk/pull/23691). 

Do we just revert this line like https://github.com/dotnet/sdk/pull/23691/commits/fcde58701af571bb92bd8cf4c7fefa29527b22fa?